tin/templates/courses/home.html

```diff
@@ -88,6 +112,33 @@ <h2>To Do</h2>
     </ul>
   {% endif %}
 
+  {% if expired_courses %}
+    <div class="dropdown">
+      <h2 id="archive-courses-toggle""
+        onclick="toggle()">Archived Courses <span id="arrow" style="font-weight:normal"></span></h2>
+      <div class="dropdown-content">
+        <ul id="archive-list">
+          {% for course in expired_courses %}
+            <li>
+              <div class="left">
+                <a href="{% url 'courses:show' course.id %}">{{ course.name }}</a>
+                {% if course in courses_with_unsubmitted_assignments %}
+                  <br>
+                  <span class="italic">Unsubmitted assignments</span>
+                {% endif %}
+              </div>
+              <div class="right">
+                Teachers: {{ course.get_teacher_str }}<br>
+                {{ course.students.count }} student{{ course.students.count | pluralize }}
+              </div>
+            </li>
+          {% endfor %}
+        </ul>
+      </div>
+    </div>
+  {% endif %}
+
+
```

Comment:
I again think this section should be moved up - right below the courses list (and above "Due in the next week" and "To Do"). Any reason you think it's not a good idea?